.vscode/tasks.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I don't know if it makes sense to be checking in vscode specific files. Do we have to do this?

Either way, let's add vscode files to .gitignore to avoid accidental commits, even if we want to keep this initial check-in:
https://github.com/Azure/azure-sdk-for-c/blob/d06072da51d0f026b17a07c2a33b93da54d82277/.gitignore#L339-L340

Reply:
+1, I don't think it's a good idea to check in these vscode specific config files either. After all, not everybody uses VScode, and there are many configurations different from person to person, for example, not everybody installs gcc to `/usr/bin/`